src/components/dropdown-menu/DropdownMenuItemButtonDownload.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { IconDownload } from '..';
+import {
+  DropdownMenuItemButtonBase,
+  DropdownMenuItemButtonBaseProps,
+} from './DropdownMenuItemButtonBase';
+
+interface Props
+  extends Omit<DropdownMenuItemButtonBaseProps, 'isActivable' | 'isActive' | 'prefix'> {}
+
+export function DropdownMenuItemButtonDownload(props: Readonly<Props>) {
+  return (
+    <DropdownMenuItemButtonBase
```

Comment:
The download item would actually be a link with some special attributes. I see this in SC:
```
interface ItemDownloadProps extends ListItemProps {
  download: string;
  href: string;
  type?: string;
}

export function ItemDownload(props: Readonly<ItemDownloadProps>) {
  const { children, className, download, href, innerRef, type, ...liProps } = props;
  return (
    <li className="dropdown-menu-download" ref={innerRef} role="none" {...liProps}>
      <ItemDownloadStyled
        className={className}
        download={download}
        href={href}
        rel="noopener noreferrer"
        role="menuitem"
        target="_blank"
        type={type}>
        {children}
      </ItemDownloadStyled>
    </li>
  );
}
```

Reply:
cutting this branch for now, the special behavior will be implemented later